graphviz2drawio/graphviz2drawio.py

```
@@ -22,7 +22,7 @@ def convert(graph_to_convert: AGraph | str | IO, layout_prog: str = "dot") -> st
     graph_nodes: dict[str, dict] = {n: n.attr.to_dict() for n in graph.nodes_iter()}
 
     svg_graph = graph.draw(prog=layout_prog, format="svg")
-
+    
```

Ruff (0.8.0)

Blank line contains whitespace

Remove whitespace from blank line

(W293)